tests_e2e/orchestrator/runbook.yml

```diff
@@ -56,7 +56,6 @@ variable:
       - no_outbound_connections
       - publish_hostname
       - recover_network_interface
-      - cgroup_v2_enabled
```

Comment:
Removed cgroup_v2 specific test case, now merged to existing agent_cgroups, ext_cgroups